ContextDesktop/src/main/java/com/github/migi_1/Context/model/MainEnvironment.java

```diff
@@ -279,7 +298,11 @@ private void initSpatials() {
         for (Path path : levelGenerator.getPathPieces(COMMANDER_LOCATION)) {
             addDisplayable(path);
         }
-
+        for (PlatformPosition position : PlatformPosition.values()) {
+        	Carrier carrier = createCarrier(position);
+        	addEntity(carrier);
+        }
+        
```

Comment:
Not needed, as four players are REQUIRED to connect. The only reason that we can start the game without is because we're still in the development phase, and getting four people to connect is cumbersome.